<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Localizar Funcion√°rio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(10, 10, 10);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="login_area">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>700</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(35, 35, 35);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>410</y>
            <width>231</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
}

QPushButton:hover{
	
	border: 2px solid rgb(70,70,70);
	border-radius: 5px;
}

QPushButton:pressed{
	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255,165,24);
	color: 5px;
	color: rgb(35,35,35);
}</string>
          </property>
          <property name="text">
           <string>TROCAR SENHA</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>330</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>530</y>
            <width>181</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
}

QPushButton:hover{
	
	border: 2px solid rgb(70,70,70);
	border-radius: 5px;
}

QPushButton:pressed{
	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255,165,24);
	color: 5px;
	color: rgb(35,35,35);
}</string>
          </property>
          <property name="text">
           <string>EXCLUIR</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>530</y>
            <width>181</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
}

QPushButton:hover{
	
	border: 2px solid rgb(70,70,70);
	border-radius: 5px;
}

QPushButton:pressed{
	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255,165,24);
	color: 5px;
	color: rgb(35,35,35);
}</string>
          </property>
          <property name="text">
           <string>VOLTAR</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_7">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>70</y>
            <width>181</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
}

QPushButton:hover{
	
	border: 2px solid rgb(70,70,70);
	border-radius: 5px;
}

QPushButton:pressed{
	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255,165,24);
	color: 5px;
	color: rgb(35,35,35);
}</string>
          </property>
          <property name="text">
           <string>Localizar</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>50</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="login">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>70</y>
            <width>151</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	color: rgb(100,100,100);
	background-color: rgb(30, 30,30);
}

QLineEdit:hover{
	border: 2px solid rgb(55,55,55);

}

QLineEdit:focus{
	border: 2px solid rgb(255,207,0);
	color: rgb(200,200,200);
}</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>90</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>LOGIN:</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>170</y>
            <width>311</width>
            <height>241</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200,200,200);
color: rgb(0, 0, 0);
border-radius: 5px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nome</string>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Login</string>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="pushButton_8">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>470</y>
            <width>381</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
}

QPushButton:hover{
	
	border: 2px solid rgb(70,70,70);
	border-radius: 5px;
}

QPushButton:pressed{
	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255,165,24);
	color: 5px;
	color: rgb(35,35,35);
}</string>
          </property>
          <property name="text">
           <string>TODOS OS FUNCION√ÅRIOS</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton10">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>410</y>
            <width>31</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pdf.png</normaloff>pdf.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>410</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
}

QPushButton:hover{
	
	border: 2px solid rgb(70,70,70);
	border-radius: 5px;
}

QPushButton:pressed{
	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255,165,24);
	color: 5px;
	color: rgb(35,35,35);
}</string>
          </property>
          <property name="text">
           <string>SALVAR</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>140</y>
            <width>231</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottom">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(20, 20, 20);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Anderson Peixoto 1.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>login</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="file_rc.qrc"/>
 </resources>
 <connections/>
</ui>
